### PR TITLE
Hotfix on release version

### PR DESCRIPTION
**Pull request by l-vincent-l**
for authentication and embedded_type

### CODE REVIEW

**Comment by l-vincent-l** [comment]
Jenkins, test this please

File: source/jormungandr/jormungandr/instance_manager.py [hunk 1]
```diff
@@ -201,7 +201,8 @@ def key_of_id(self, object_id, only_one=True):
         if ptobject:
             instances = ptobject.instances()
             if len(instances) > 0:
-                available_instances = [i.name for i in instances if authentication.has_access(i, abort=False)]
+		user = authentication.get_user()
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/instance_manager.py]: you have tabs here

File: source/jormungandr/jormungandr/instance_manager.py [hunk 2]
```diff
@@ -232,7 +233,8 @@ def key_of_coord(self, lon, lat, only_one=True):
                     raise RegionNotFound(custom_msg="technical problem, impossible "
                                                     "to find region {r} in jormungandr database"
                                          .format(r=key))
-                if authentication.has_access(jormun_instance, abort=False):  #TODO, pb how to check the api ?
+		user = authentication.get_user()
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/instance_manager.py]: you have tabs here

File: source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py [hunk 3]
```diff
@@ -116,7 +116,8 @@ def add_id(objects, region, type_=None):
                     if 'embedded_type' in objects.keys() and\
                         (objects['embedded_type'] == 'address' or
                          objects['embedded_type'] == 'poi' or
-                         objects['embedded_type'] == 'administrative_region'):
+                         objects['embedded_type'] == 'administrative_region') and\
+			objects['embedded_type'] in objects:
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py]: you have tabs here

**Pull request merged**